README.md

```diff
@@ -66,39 +65,49 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
     ./install.sh
     ```
 
-4. Apache configuration and restart 
-LORIS requires Apache's mod_rewrite module to rewrite its URLs. Enable this module, then restart Apache: 
-
+4. Run the makefile (use `make dev` if you are setting up a development sandbox)
     ```
-    sudo a2enmod rewrite
-    sudo service apache2 reload
+    cd /var/www/$projectname
+    make
     ```
 
-5. Go to http://localhost/installdb.php and follow the instructions to finalize LORIS installation.
-
-    _Note_: Apache config files will be installed as *.conf, per Ubuntu 14.04. If running an earlier version of Ubuntu, rename these files, then run the following commands. After, restart Apache.
+5. Apache configuration
 
-
-    ```
+If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:
+    
+    sudo a2enmod rewrite
```

Comment:
These commands should be run as `root` or user with `sudo` privileges, not as `lorisadmin`. This should be done before the command `su - lorisadmin`

Reply:
I thought that the lorisadmin user was to be added to the sudo group

Reply:
There are two ways to approach this:

1) `root` or equivalent can be used during the install process to do these specific steps
2) `lorisadmin` can be added to sudoers group

I think 2) is a worse approach because we should be using the least number of permissions possible for LORIS and there isn't a very good reason that a user created just for the LORIS app should have god-mode privileges on the system. That sort of defeats the point of having users create `lorisadmin` user on the system just for this webapp. They might as well use `root`.

Reply:
Then step 1 of install instruction in README.md as to be removed.

Reply:
I wrote about what I feel is the best way to do this in #5297 which would require more substantive changes. My original comment can be postposed for further refactoring another time.